app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -29,4 +35,15 @@ def from_omniauth(auth)
       u.save
     end
   end
+
+  def from_omniauth_discord(auth)
```

Comment:
change the name of from_omniauth and remove some duplications from from_omniauth_* methods

Reply:
done